src/main/java/org/opentripplanner/updater/bike_rental/BikeRentalUpdater.java

```diff
@@ -54,6 +54,8 @@
 
     private int timeToLiveMinutes;
 
+    private String networkName = "default";
```

Comment:
What is the difference with private field `network` above?

Reply:
I also had to do a long double take when I saw this too. The `network` field could be thought of as `compatibleNetworks` as in a comma-separated list of strings that identify the compatible networks where a vehicle can be dropped off at. The `networkName` here is the name of the specific network of this particular bike rental updater. I will strongly advocate for rewording of this when updating to GBFS 2.2 and even a check on whether this is even needed anymore.